plane/src/typed_unix_socket/client.rs

```diff
@@ -9,32 +10,22 @@ use tokio::{
     net::UnixStream,
     sync::{broadcast, oneshot},
 };
-use uuid::Uuid;
 
 #[derive(Clone)]
-pub struct TypedUnixSocketClient<ClientMessageType, ServerMessageType, RequestType, ResponseType>
+pub struct TypedUnixSocketClient<MessageToServer, MessageToClient>
```

Comment:
I find `{Client,Server}MessageType` ambiguous because it's not clear whether a client message is a message sent by or to the client. I like the convention `MessageToX` or `MessageFromX` to make the distinction obvious.